scripts/meetingbaas.py

```
@@ -9,38 +9,55 @@
 from dotenv import load_dotenv
 from pipecat.adapters.schemas.function_schema import FunctionSchema
 from pipecat.adapters.schemas.tools_schema import ToolsSchema
-from pipecat.vad import SileroVADAnalyzer, VADParams
-from pipecat.frames.frames import LLMMessagesFrame
+from pipecat.audio.vad.silero import SileroVADAnalyzer, VADParams
+from pipecat.frames.frames import LLMMessagesFrame, TTSAudioRawFrame, TextFrame
```

Ruff (0.11.9)

`pipecat.frames.frames.TTSAudioRawFrame` imported but unused

Remove unused import: `pipecat.frames.frames.TTSAudioRawFrame`

(F401)